### PR TITLE
Add zoom component to the canvas

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
_Can be tested locally_
## QA
- Add something to the canvas
- Click the plus and minus buttons and check they zoom in and out
- There is no design for max'ed out values and not sure there required
- Check it matches the design [1]

[1] https://cloud.githubusercontent.com/assets/9105999/18986186/ac97e32a-86f3-11e6-8474-0f1282af4b0f.png

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6033/
Test PASSed.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1226,6 +1226,45 @@ YUI.add('juju-gui', function(Y) {
...
+        zoomOutCanvas={this._zoomOutCanvas.bind(this)} />,
+        document.getElementById('zoom-container'));
+    },
+
+    /**
+      Fires the zoom function with an increased zoom value
+
+      @method _zoomInCanvas
+    */
+    _zoomInCanvas: function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: this method and the one below don't appear to have any use in app.js, only in the component. Can you move them into the component and pass in `topo` to the component.

**Comment by makyo** [comment]
QA okay

File: jujugui/static/gui/src/app/app.js (hunk 1 above)
Comment (anthonydillon) [inline jujugui/static/gui/src/app/app.js]: Good call, done.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6039/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6040/
Test PASSed.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**